CardDictionaries/ArcaneRising/ARCWizard.php

```diff
@@ -404,7 +417,11 @@ function ActionsThatDoArcaneDamage($cardID)
       case "DYN203": case "DYN204": case "DYN205":
       case "DYN206": case "DYN207": case "DYN208":
         return true;
-      case "HVY252": 
+      case "HVY252":
+        return true;
+      case "ROS207": case "ROS176": case "ROS189": case "ROS198": case "ROS201": return True;
+      case "ROS208": case "ROS177": case "ROS190": case "ROS199": case "ROS202": return True;
+      case "ROS209": case "ROS178": case "ROS191": case "ROS200": case "ROS203": return True;
```

Comment:
I see what you are trying to do here sorting wise but I think it would be more readable if the cards with the same name were just on the same line and were sorted by set number ascending again.